clean_helpers/sentence_splitter.py

```diff
@@ -0,0 +1,76 @@
+import subprocess
+import os
+import torch
+import stanza
+from stanza_batch import batch
+from indicnlp import common
+from indicnlp.tokenize import sentence_tokenize
+import nltk
+from nltk.tokenize import sent_tokenize
+
+
+def build_nltk_splitter(lang):
+    nltk.download("punkt")
+    lang_to_punkt = {
+        "en": "english",
+        "fr": "french",
+        "pt": "portuguese",
+        "es": "spanish"
+    }
+    
+    def splitter(examples):
+        split_texts = ["\n".join(sent_tokenize(text, language=lang_to_punkt[lang])) for text in examples["text"]]
+        return {**examples, "text": split_texts }        
+    return splitter
+
+
+def build_stanza_splitter(lang, batch_size=32):
+    if len(lang)==3:
+        lang.replace("zh", "zh-han") # zhs-> zh-hans, zht -> zh-hant
+    stanza.download(lang, logging_level="WARNING")
+    nlp = stanza.Pipeline(lang, logging_level="WARNING", processors='tokenize',
```

Comment:
```suggestion
    tokenizer = stanza.Pipeline(lang, logging_level="WARNING", processors='tokenize',
```

Reply:
Hum so this has to run in a single process? If you run multiple process is this serializable correctly? Should we not create an instance for each process?

Reply:
Yep, it should be run on a single process and you could probably create a new instance per process.

Reply:
Have you tried running this with `num-proc > 1`?